Makefile

```diff
@@ -279,14 +279,20 @@ GLOBAL_ASM_RACING_O_FILES = $(foreach file,$(GLOBAL_ASM_RACING_C_FILES),$(BUILD_
 
 # detect prefix for MIPS toolchain
 ifneq ($(CROSS),)
```

Comment:
why you rearrange the priority ? mips64-elf- it's the version of libdragon no ? on your os it work only with a specific version ?

Reply:
It uses the same arragement as in this fork of SM64
https://github.com/CrashOveride95/ultrasm64/blob/master/Makefile#L329

Reply:
yes but why ?

Reply:
My WSL2 environment has a different N64 toolchain, I can build IDO just fine but for GCC i need those extra cross checks

Reply:
ok ok it's fine for me also